docs/conversation.md

```diff
@@ -115,7 +115,6 @@
 ## Creating a new conversation
 
 *Note:* Creating a conversation as a child breakout room, will automatically set the lobby when breakout rooms are not started and will always overwrite the room type with the parent room type. Also, moderators of the parent conversation will be automatically added as moderators.
-
```

Comment:
this needs to be readded (otherwise the docs list breaks), try to run `mkdocs serve` locally
![grafik](https://github.com/user-attachments/assets/3ae9cda3-e610-4ac3-8093-0d2342d39816)